src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const objStyle = sourceString.split(';');
+
+  return objStyle.reduce((acc, item) => {
+    const prop = item.split(':');
```

Comment:
```suggestion
    const [key, value] = item.split(':');
```

You can use array destructuring